Mathlib/Tactic/ImportDiff.lean

```diff
@@ -0,0 +1,33 @@
+/-
+Copyright (c) 2025 Paul Lezeau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Paul Lezeau
+-/
+import Batteries.Data.List.Basic
+import Mathlib.Init
+
+/-! # The `#import_diff` command
+
+This file implements the `#import_diff` commmand, which is useful for debugging large import changes
+by determining the transitive imports added by a given import.
+
+-/
+
+open Lean Elab Meta
+
+/-- `#import_diff foo` computes the new transitive imports that are added to a given file when
+module `foo` is added to the set of imports of the file. -/
+elab "#import_diff" n:ident : command => do
+  let name := n.getId
```

Comment:
How about adding some form of validation that the module name provided actually exists?

Maybe using something like [Lean.findOLean](https://leanprover-community.github.io/mathlib4_docs/Lean/Util/Path.html#Lean.findOLean)?

Try to test how gracefully it fails if you give it a module that does not have `olean`s (which presumably should fail, since you probably do not want this command to start building lean files) and whether it works for imports in, e.g., Batteries when you are in Mathlib.